app/services/publishing_api_payload.rb

```diff
@@ -34,6 +34,15 @@ def payload
 
 private
 
+  def links
+    associations = document.associations
+    if associations.include?("primary_publishing_organisation") && !associations.include?("original_primary_publishing_organisation")
+      associations["original_primary_publishing_organisation"] = associations["primary_publishing_organisation"]
+      document.update(associations: associations)
```

Comment:
I don’t think we should be changing the document in what looks like an accessory method. We don’t actually need to change it either - the original org is something we could derive each time we send the payload.